pages/bloghome.html

```diff
@@ -0,0 +1,155 @@
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>A yr of Code Blog | Read me </title>
+      <link rel="stylesheet" href="styles/style.css">
+      <link rel="stylesheet" media="screen and (max-width: 768px)" href="styles/mobile.css">
+      <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+   </head>
+   <body>
+      <div class = "container">
+         <!-- Navbar and Page Menu -->
+         <div class="header box">
+            <div class="logo">
+               <a class="logo" href="bloghome.html">
+               <img src="images/bloglogo.png.png" alt="Homepage">
+               </a>
+               <h1>
+                  A year of Code Blog
+               </h1>
+               <ul class="listnav">
+                  <li><a href="#">About</a></li>
+               </ul>
+               <ul class="listnav">
+                  <li><a href="blog.html"> Most Current Blog Post</a></li>
+               </ul>
+            </div>
+         </div>
+      </div>
+      <!-- Blog showcase/most current posting-->
+      <div>
+         <div class="showcasebox">
+            <div class="showcasecontent">
+               <h1><span>Blog Post title</span></h1>
+               <p><span>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Incidunt quidem est itaque doloremque 
+                  facere culpa velit quo a assumenda. Repudiandae id aliquid vel eaque praesentium architecto 
+                  distinctio quis in pariatur.</span>
+               </p>
+               <a href="blog.html" class="btn"> Read More</a>
+            </div>
+         </div>
+         <!-- Old Blog Posts-->
+         <section class="postcontainer">
+            <h2 class="mostread"><span>Most Read Posts</span></h2>
+            <div class="posts">
+               <article class="card">
+                  <div class="order">
+                     <img src="images/blog1.jpg" alt="woman">
+                  </div>
+                  <div>
+                     <h3>
+                        <a href="#"> Lorem impsum dolor sit</a>
+                     </h3>
+                     <p>
+                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Impedit quis similique nemo omnis, 
+                        perspiciatis laborum soluta libero nihil vitae doloremque facere, animi laboriosam, tenetur 
+                        ipsum qui illo neque! Vel, fugiat!
+                     </p>
+                  </div>
+               </article>
+               <article class="card">
+                  <div class="order">
+                     <img src="images/blog2.jpg" alt="woman">

```

Comment:
Make sure that the alt tag describes the image.

Reply:
It's a good idea to resize images to be lower resolution. If the image data is much larger than the size it will be displayed at, this causes the visitor to consume more data than necessary, and slows down displaying the page (large images are harder for the browser to display and take some time to resize).